lms/templates/courseware/search.html

```diff
@@ -0,0 +1,115 @@
+<%page expression_filter="h"/>
+<%inherit file="../main.html" />
+<%def name="online_help_token()"><% return "courseinfo" %></%def>
+<%namespace name='static' file='../static_content.html'/>
+<%!
+from django.utils.translation import ugettext as _
+
+from courseware.courses import get_course_info_section, get_course_date_summary
+
+from openedx.core.djangoapps.self_paced.models import SelfPacedConfiguration
+from openedx.core.djangolib.markup import HTML, Text
+%>
+
+<%block name="pagetitle">${_("{course_number} Search").format(course_number=course.display_number_with_default)}</%block>
+
+<%block name="headextra">
+<%static:css group='style-course-vendor'/>
+<%static:css group='style-course'/>
+</%block>
+
+<%include file="/courseware/course_navigation.html" args="active_page='bookmarks'" />
+
+<%static:require_module_async module_name="js/courseware/toggle_element_visibility" class_name="ToggleElementVisibility">
+        ToggleElementVisibility();
+</%static:require_module_async>
+<%static:require_module_async module_name="js/courseware/course_home_events" class_name="CourseHomeEvents">
+        CourseHomeEvents();
+</%static:require_module_async>
+
+<%block name="bodyclass">view-in-course view-course-info ${course.css_class or ''}</%block>
+
+<script type="text/javascript">
+  function handleSearch() {
+    var searchQuery = $('#searchQuery').val();
+    var recentlyViewedFilter = $('#recentlyViewedFilter').hasClass('active');
+    var bookmarkedFilter = $('#bookmarkedFilter').hasClass('active');
+
+    var search = 'query=' + searchQuery;
+    search += '&recentlyViewed=' + recentlyViewedFilter;
+    search += '&bookmarked=' + bookmarkedFilter;
+
+    window.location.search = search;
+  }
+</script>
```

Comment:
Is there a reason why script tag is opening and closing for each function? or is this legacy.... :smile: